constants/SkillSelect/Skills.js

```diff
@@ -0,0 +1,49 @@
+export default {
```

Comment:
sorry, can we move this to `constants/Skills.js`? there will be other components and screens that will need this list, not just the skill select component.

Reply:
Sounds good